article/app/layout/LiveBlogs.scala

```diff
@@ -0,0 +1,35 @@
+package layout
```

Comment:
Is the `layout` directory a bit confusing? It's a fairly loaded name, I thought it was to do with facia-like layout. maybe move this one to `article/app/liveblog/blockformat.scala`?